eth-contracts/README.md

```diff
@@ -0,0 +1,15 @@
+# CKBChain
+a ckb light client and spv prover on ethereum
+
+# TokenLocker
+TokenLocker is linked to the fungible token on Ethereum side and mintable fungible
+token on CKB side, it also links to the prover that it uses to unlock the tokens.
+
+## Quick Start
```

Comment:
```suggestion
The ethereum contracts component of forece-bridge-eth, mainly contains `CKBChain` and `TokenLocker`.

# CKBChain
CKBChain is a ckb light client and spv prover.

# TokenLocker
TokenLocker locks erc-20 token or eth, and emits events, which can be used to mint sUDT token on CKB. In reverse, it handle bridge message(sUDT burned) from CKB, and unlock erc-20 token or eth if the message is verified.

# Quick Start
```

Reply:
There is usually only one level 1 title (`#`) in a doc.